com.unity.render-pipelines.high-definition/Editor/VFXGraph/Shaders/Templates/Mesh/PassDepthForwardOnly.template

```diff
@@ -0,0 +1,17 @@
+Pass
+{		
+	Tags { "LightMode"="DepthForwardOnly" }
+
+	ZWrite On
+	Blend Off
+	
+	HLSLPROGRAM
+	#define VFX_PASSDEPTH VFX_PASSDEPTH_ACTUAL
+	#pragma multi_compile _ WRITE_NORMAL_BUFFER
+    #pragma multi_compile _ WRITE_DECAL_BUFFER
```

Comment:
You mixed space & tab here ⬆️, not a big deal (it was probably already the case in the source file)